front-end/src/Components/Home.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { buttonHandlerDelete, buttonHandlerSignOut, buttonHandlerUpdate } from './Fucntions';
+import { useNavigate } from 'react-router-dom';
+
+export default function Home(props) {
+
+  const navigate = useNavigate();
+  return (
+    <div className='border-cyan-600 border-2'>
+      <h1>First Name: {props?.schemaFirstName}</h1>
+      <h1>Last Name: {props?.schemaLastName}</h1>
+      <p>Age: {props?.schemaAge}</p>
+      <p>Gender: {props?.schemaGender}</p>
+      <p>Profession: {props?.schemaProfession}</p>
+      <br />
```

Comment:
Try Avoiding BR, using one br is good or using two is good but using alot of br one after another is not good

Reply:
okay I will take care. I only wrote this for development and just did not remove these. I will take care that this never happens. My major focus was on the backend